src/30_Advanced/Integrating_Videos_in_AMP%3A_an_Overview.html

```diff
@@ -0,0 +1,135 @@
+  </amp-video>
+  <!-- #### Instagram -->
+  <!--
+    Use `amp-instagram` for videos on Instagram.
+    Find more examples [here](https://ampbyexample.com/components/amp-instagram/)
+  -->
+  <amp-instagram
+      data-shortcode="BCtYW7mhRV8"
+      width="1"
+      height="1"
+      layout="responsive">
+  </amp-instagram>
+  <!-- #### Vine -->
+  <!--
+    Use `amp-vine` for vine embed.
+    Find more examples [here](https://ampbyexample.com/components/amp-vine/)
+  -->
+  <amp-vine width="400" height="400"
+    data-vineid="MdKjXez002d">
+  </amp-vine>
+  <!-- #### Youtube -->
+  <!--
+    Use `amp-youtube` for youtube videos.
+    Find more examples [here](https://ampbyexample.com/components/amp-youtube/)
+  -->
+  <amp-youtube width="480" height="270"
+                           layout=responsive
+                           data-videoid="lBTCB7yLs8Y" >
+  </amp-youtube>
+  <!-- #### Vimeo -->
+  <!--
+    Use `amp-vimeo` for vimeo videos.
+    Find more examples [here](https://ampbyexample.com/components/amp-vimeo/)
+  -->
+  <amp-vimeo
+      data-videoid="27246366"
+      layout="responsive"
+      width="16" height="9">
+  </amp-vimeo>
+  <!-- #### Dailymotion -->
+  <!--
+    Use `amp-dailymotion` for dailymotion videos.
+    Find more examples [here](https://ampbyexample.com/components/amp-dailymotion/)
+  -->
+  <amp-dailymotion
+    data-videoid="x2m8jpp"
+    layout="responsive"
+    data-ui-highlight="FF4081"
+    width="480" height="270">
+  </amp-dailymotion>
+  <!-- #### Twitter -->
+  <!-- 
+    Use `amp-twitter` for twitter videos.
+    Find more examples [here](https://ampbyexample.com/components/amp-twitter/)
+  -->
+  <amp-twitter width="375" height="472"
+    layout="responsive"
+    data-tweetid="707569406105092096">
+  </amp-twitter>
+  <!-- #### Facebook -->
+  <!-- 
+    Use `amp-facebook` for facebook videos.
+    Find more examples [here](https://ampbyexample.com/components/amp-facebook/)
+  -->
+  <amp-facebook width="552" height="310"
+                    layout="responsive"
+                    data-href="https://www.facebook.com/zuck/videos/10102509264909801/">
+  </amp-facebook>
+  <!-- #### Brightcove -->
+  <!-- 
+    Use `amp-brightcove` for brightcove videos.
+    Find more examples [here](https://ampbyexample.com/components/amp-brightcove/)
+  -->
+  <amp-brightcove
+                  data-account="906043040001"
+                  data-video-id="1401169490001"
+                  data-player-id="180a5658-8be8-4f33-8eba-d562ab41b40c"
+                  layout="responsive" width="480" height="270">
+  </amp-brightcove>
+  <!-- #### Iframe -->
+  <!-- 
+    Use `amp-iframe` for native videos not supported in Amp.
+    Find more examples [here](https://ampbyexample.com/components/amp-iframe/)
+  -->
+  <amp-iframe width="500" height="281"
+                          layout="responsive"
+                          sandbox="allow-scripts allow-same-origin allow-popups allow-popups-to-escape-sandbox"
```

Comment:
change the sandbox args to: 'sandbox="allow-scripts allow-same-origin allow-popups' and add the allowfullscreen attribute (see my PR for amp-iframe).